### ISSUE TITLE
Folding expressions missing parenthesis

### ISSUE THREAD

**Issue by jonathanpoelen** [andreasfertig/cppinsights#264]
Similar to #91, but for other operators.

```cpp
auto f = [](auto... i) { return (.../i); }; // (__i0 / __i1 / __i2)
auto g = [](auto... i) { return (i/...); }; // (__i0 / __i1 / __i2) also
f(1,2,3);
g(1,2,3);
```

https://cppinsights.io/lnk?code=aW50IG1haW4oKQp7CiAgYXV0byBmID0gW10oYXV0by4uLiBpKSB7IHJldHVybiAoLi4uL2kpOyB9OwogIGF1dG8gZyA9IFtdKGF1dG8uLi4gaSkgeyByZXR1cm4gKGkvLi4uKTsgfTsKICBmKDEsMiwzKTsKICBnKDEsMiwzKTsKfQ==&insightsOptions=cpp2a&std=cpp2a&rev=1.0

**Comment by andreasfertig** [comment]
Hello @jonathanpoelen,

thank you for discovering and reporting this. I probably was to conservative with the fix in #91. 
I will push a fix probably by today.

Andreas

**Comment by andreasfertig** [comment]
Looks like there are multiple issues with the CI builds. With that, I will not be able to provide a fix today.

Andreas
